src/gc.c

```diff
@@ -13,6 +13,11 @@
 #include "numeric_index.h"
 #include "tag_index.h"
 #include "config.h"
+#include <unistd.h>
+#include <sys/wait.h>
+#include <stdbool.h>
+
+uint32_t gcRevisionId = 0;
```

Comment:
This will cause linking issues. Either declare as extern in a header, or declare as static and limit to file-only.

Reply:
Its an unused var, I removed it.